seungjong/app.js

```diff
@@ -112,6 +112,68 @@ app.get('/users/:userId/posts', async (req, res, next) => {
 });
 
 
+app.patch('/users/:userId/:postId', async (req, res, next) => {
+    const { userId, postId } = req.params;
+    const { userName, title, content } = req.body;
+
+    await dataSource.query(`    
+        UPDATE posts
+        SET
+        content = ?
+        WHERE id = ?     
+        `, [content, postId]
+        )
+    
+    const result = await dataSource.query(`
+        SELECT
+        u.id,
+        u.name,
+        p.id,
+        p.title,
+        p.content
+        FROM users as u
+        JOIN posts as p
+        ON u.id = p.user_id
+        WHERE p.id = ?;
+    `, [postId]);
+
+    res.status(200).json({ data: {
+        userId : userId,
+        userName: result[0].name,
+        postingId : postId,
+        postingTitle: result[0].title,
+        postingContent: content
+    } });
+})
+
+
+app.delete('/users/:userId/:postId/postDelete', async(req, res, next) => {
+    const {userId, postId} = req.params;
+    await dataSource.query(`
+        DELETE FROM posts as p
+        WHERE p.id = ?
+        AND p.user_id = ?;
+    `, [postId, userId]);
+    res.status(200).json({message: "postingDeleted"});
```

Comment:
```suggestion
    res.status(204).send()
```
delete method에 보다 더 적합한 status code도 있습니다. 다만 해당 204 status code로 응답을 할 때에는 body에 무언가를 담아서 response 할 수 없기 때문에, `send()` 메소드 호출로 응답을 보내주기만 해도 됩니다.